src/components/LookerLink.svelte

```diff
@@ -0,0 +1,79 @@
+<script>
+  import { tooltip as tooltipAction } from '@graph-paper/core/actions';
+
+  import LookerLogo from './icons/LookerLogo.svelte';
+
+  export let product;
+  export let variants;
+  export let sendInPings;
+  export let channel;
+
+  let variant;
+  let lookerURL;
+  let helpText;
+
+  $: {
+    variant = variants && variants.find((el) => el.channel === channel);
+
+    lookerURL =
+      variant &&
+      variant.etl.ping_data[sendInPings].looker &&
+      variant.etl.ping_data[sendInPings].looker.metric.url;
+
+    helpText =
+      (lookerURL && 'Explore this metric in Looker.') ||
+      (product === 'firefox' &&
+        'Looker is only available in Glean applications.') ||
```

Comment:
I might suggest putting this inside the HTML/presentation instead of making it a reactive assignment.

```suggestion
        'Looker is only available for Glean applications.') ||
```